services/api/src/mocks.js

```diff
@@ -547,8 +525,9 @@ mocks.Query = () => ({
   userCanSshToEnvironment: () => mocks.Environment(),
   deploymentByRemoteId: () => mocks.Deployment(),
   taskByRemoteId: () => mocks.Task(),
-  allProjects: () => new MockList(9),
+  allProjects: () => new MockList(600),
```

Comment:
As there quite a few other mocks that use random values here I thought it makes sense to do that same. Maybe it would be best to have a few different values here to mock displays of different sizes though.